hack/staging-import-restrictions.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/apimachinery",
+    "allowedImports": [
+      "k8s.io/kube-openapi"
+    ]
+  },
+  {
+    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/api",
+    "allowedImports": [
+      "k8s.io/apimachinery"
+    ]
+  },
+  {
+    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/kube-gen",
```

Comment:
OK. Cool. Not even sure we need ignored subtrees for that, do we? This should work:

```json
  {
    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/kube-gen",
    "allowedImports": [
      "k8s.io/gengo",
      "k8s.io/kube-openapi"
    ]
  },
  {
    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/kube-gen/test",
    "allowedImports": [
      "k8s.io/apimachinery",
      "k8s.io/client-go",
      "k8s.io/gengo",
      "k8s.io/kube-openapi"
    ]
  },
```